hazelcast/src/main/java/com/hazelcast/internal/eviction/EvictionPolicyType.java

```diff
@@ -29,8 +29,17 @@
     /**
      * Least Frequently Used
      */
-    LFU
+    LFU,
 
-    // TODO Maybe another "CUSTOM" type for user defined eviction policies
+    /**
+     * Picks a random entry
+     */
+    RANDOM,
+
+    /**
+     * Doesn't evict entries (will not add new entries to the Near Cache when it's full)
+     */
+    NONE,
```

Comment:
I think I managed to add the old behavior of `NONE` eviction which was:
```
public void put(Data key, Object value) {
    fireCacheExpiration();
    if (evictionPolicy == NONE && cache.size() >= maxSize) {
        return;
    }
```
So if there is no expiration, no new entries should be added to the Near Cache, since no eviction will be triggered.